agent/api/task/task.go

```diff
@@ -301,8 +303,25 @@ func TaskFromACS(acsTask *ecsacs.Task, envelope *ecsacs.PayloadMessage) (*Task,
 	} else if task.GetDesiredStatus() == apitaskstatus.TaskStopped && envelope.SeqNum != nil {
 		task.StopSequenceNumber = *envelope.SeqNum
 	}
+	// search the service connect attachment from acsTask.Attachments
+	if acsTask.Attachments != nil {
```

Comment:
Could you add test for this?

Reply:
`TestTaskServiceConnectAttachment()` and `TestTaskWithoutServiceConnectAttachment()` are added in commit 3d8d95406e74fc25d3c00d384a11cd5cd4ae55f9. Please take a look on `agent/api/task/task_test.go`. Thanks.